server/games/game.py

```diff
@@ -614,11 +618,11 @@ async def validate_game_settings(self):
             await self.mark_invalid(ValidityState.FFA_NOT_RANKED)
             return
         valid_options = {
-            "AIReplacement": ("Off", ValidityState.HAS_AI_PLAYERS),
```

Comment:
Oh wow I didn't realize that there were even more words for true/false used already 😅